src/PopoverMenuItem.jsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import { Link } from 'react-router';
+import ReactDOM from 'react-dom';
+import cx from 'classnames';
+
+/**
+ * @module PopoverMenuItem
+ */
+class PopoverMenuItem extends React.Component {
+
+	componentDidUpdate() {
+		if (this.selectedItem) {
```

Comment:
Might be a little nitpicky but; I would maybe change this up slightly, instead of having logic in the jsx to check if the menuItem is selected instead i would perhaps do that here.

so something like

```
if (this.prop.isSelected) {
  ReactDOM.findDOMNode(this.myLink).focus();
}
```

and may a reference to `myLink`